web/annotation_layer_builder.js

```diff
@@ -120,17 +120,22 @@ class AnnotationLayerBuilder {
 
     // Create an annotation layer div and render the annotations
     // if there is at least one annotation.
-    this.div = document.createElement("div");
-    this.div.className = "annotationLayer";
-    this.pageDiv.append(this.div);
+    const div = (this.div = document.createElement("div"));
+    div.className = "annotationLayer";
+    this.pageDiv.append(div);
+    this.#annotationLayer = new AnnotationLayer({
+      div,
+      accessibilityManager: this._accessibilityManager,
+      annotationCanvasMap: this._annotationCanvasMap,
+    });
```

Comment:
Why do we need to do this *unconditionally* here, as opposed to moving it down below the `this.#numAnnotations === 0` branch, the since if there's no annotations in the document then there's nothing to potentially move into the Editor?